packages/gatsby/src/schema/graphql-engine/lmdb-bundling-patch.ts

```diff
@@ -16,20 +20,25 @@ import { createRequireFromPath } from "gatsby-core-utils"
 //  - https://github.com/DoctorEvidence/lmdb-js/blob/544b3fda402f24a70a0e946921e4c9134c5adf85/open.js#L77
 // Reliance on `import.meta.url` + usage of `.replace` is what seems to cause problems currently.
 
-export default function (source: string): string {
-  let lmdbBinaryLocation
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export default function (this: any, source: string): string {
```

Comment:
These changes are necessary as lmdb can also be required from gatsby-core-utils which can lead to multiple versions. Wich leads to different paths and we want to patch all lmdb versions that need patching.